src/schemas/review_form.yaml

```diff
@@ -0,0 +1,20 @@
+type: object
```

Comment:
Score and Confidence?
[See mocks](https://peerreq-mockup.bits0rcerer.org/categories/1/1)